data-prepper-plugins/kafka-connect-plugins/build.gradle

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+plugins {
+    id 'java'
+}
+
+dependencies {
+    implementation project(':data-prepper-plugins:common')
+    implementation project(':data-prepper-plugins:kafka-plugins')
+    implementation 'org.apache.kafka:connect-runtime:3.5.1'
+    implementation 'software.amazon.awssdk:sts'
+    implementation 'software.amazon.awssdk:secretsmanager'
+    implementation 'javax.validation:validation-api:2.0.1.Final'
+    implementation libs.reflections.core
+    implementation 'io.micrometer:micrometer-core'
+    implementation 'org.eclipse.jetty:jetty-server:9.4.51.v20230217'
+    implementation 'org.eclipse.jetty:jetty-servlet:9.4.51.v20230217'
+    implementation 'org.eclipse.jetty:jetty-servlets:9.4.51.v20230217'
+    implementation 'org.eclipse.jetty:jetty-client:9.4.51.v20230217'
+    implementation ('io.confluent:kafka-schema-registry:7.5.0') {
+        exclude group: 'org.glassfish.jersey.containers', module: 'jersey-container-servlet'
+        exclude group: 'org.glassfish.jersey.inject', module: 'jersey-hk2'
+        exclude group: 'org.glassfish.jersey.ext', module: 'jersey-bean-validation'
+    }
+    runtimeOnly 'org.jetbrains.kotlin:kotlin-compiler:1.8.21'
+    // Common Debezium Connector
+    implementation 'io.debezium:debezium-api:2.3.0.Final'
+    implementation 'io.debezium:debezium-core:2.3.0.Final'
```

Comment:
It seems that Debezium and Kafka Connect require ZooKeeper. And I see it in the dependency path. How will this work when a user has multiple Data Prepper nodes?

Reply:
https://docs.confluent.io/platform/current/connect/userguide.html#distributed-mode

We make Kafka Connect run in distributed mode, it uses to make multiple nodes work together.
* group.id
* config.storage.topic
* offset.storage.topic 
* status.storage.topic

I cannot find a doc saying Kafka Connect requires Zookeeper to run. Zookeeper works with Kafka itself (not kafka connect).
Kafka Connect needs bootstrap.servers - A list of host/port pairs to use for establishing the initial connection to the Kafka cluster